readme/the-roadmap.md

```diff
@@ -0,0 +1,15 @@
+# Roadmap
+
+Here is what The Monaco Protocol Foundation is currently working on from both a technical and foundation point of view.
+
+# Monaco Protocol Foundation
```

Comment:
I would remove the 'Publish whitepaper' and 'Form official foundation entity' - just have the technical writings below.

Reply:
✅